project.secret.sample.js

```diff
@@ -1,11 +1,9 @@
 // your private key
-const private_key_list = [
-    '0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF',
-];
+const private_key_list = ["0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"];
 // your infora project id
 const infura_project_id = "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF";
 
 module.exports = {
-    infura_project_id,
-    private_key_list,
-}
+  infura_project_id,
+  private_key_list,
+};
```

Comment:
Since we use `.env` to configure our private key and infura info, this file is unuseful now.